lms/djangoapps/edraak_university/urls.py

```diff
@@ -15,7 +15,7 @@
        name='edraak_university_id',
     ),
     url(
-       r'^id/{}/success$'.format(settings.COURSE_ID_PATTERN),
+       r'^id/{}/(?P<is_student>.*)/success$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
option 2 was not valid because the success page is reached as a revers called from a different view. I was trying not add a new view because separate pages would have to be from different views. Still it was a messy solution.